xKhana/ayKhana
# config-config-#1-add_spring_basic_config

# How to Test 
1. Try to build and run `compose.yml`.
2. Open browser on `localhost:8080` and enter the password provided in the `be-khana` container log.
3. It should works fine by giving you white-label error :) .
